x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/query/ShapeQueryBuilderTests.java

```diff
@@ -179,7 +179,7 @@ public void testFromJson() throws IOException {
                 "  }\n" +
                 "}";
         ShapeQueryBuilder parsed = (ShapeQueryBuilder) parseQuery(json);
-        checkGeneratedJson(json, parsed);
+        checkGeneratedJson(json.replaceAll("envelope", "Envelope"), parsed);
```

Comment:
why not change it in the string?

Reply:
to test that I didn't break BWC when it comes to queries

Reply:
oh, so that `parseQuery` does the lowercase. got it. thanks